packages/celotool/src/lib/oracle.ts

```diff
@@ -94,6 +98,20 @@ export async function installHelmChart(
   )
 }
 
+export async function upgradeOracleChart(
+  celoEnv: string,
+  context: OracleAzureContext,
+  useFullNodes: boolean
+) {
+  await upgradeGenericHelmChart(
+    celoEnv,
+    releaseName(celoEnv),
+    helmChartPath,
+    await helmParameters(celoEnv, context, useFullNodes)
+  )
+  return
```

Comment:
nit: we can make this function non async and just have it:
```
return upgradeGenericHelmChart(
    celoEnv,
    releaseName(celoEnv),
    helmChartPath,
    await helmParameters(celoEnv, context, useFullNodes)
  )
```

Reply:
Good point. I still need to keep it async since I'm await-ing the helmParameters() call, but I updated it to just return the upgradeGenericHelmChart promise.